3rdparty/libjpeg-turbo/CMakeLists.txt

```diff
@@ -56,8 +56,7 @@ if(MSVC_IDE AND CMAKE_GENERATOR_PLATFORM MATCHES "arm64")
   set(CPU_TYPE arm64)
 endif()
 
-OCV_OPTION(ENABLE_LIBJPEG_TURBO_SIMD "Include SIMD extensions for libjpeg-turbo, if available for this platform" (NOT CV_DISABLE_OPTIMIZATION)
-  VISIBLE_IF BUILD_JPEG)
```

Comment:
Maybe we should add `BUILD_JPEG_TURBO` variable instead and check it here? Currently logic is not very obvious, i.e. in order to disable own JPEG library users must provide `BUILD_JPEG=OFF` **and** `BUILD_JPEG_TURBO_DISABLE=ON`.